ydb/docs/en/core/reference/ydb-cli/export-import/_includes/tools_restore.md

```diff
@@ -87,3 +91,9 @@ Matching schemas between the database and file system:
 ```bash
 {{ ydb-cli }} -p quickstart tools restore -p dir1/dir2 -i ~/backup_quickstart --dry-run
 ```
+
+### Recommended options for better performance
+
+```bash
+{{ ydb-cli }} -p quickstart tools restore -p . -i . --import-data --bandwidth=1GiB
```

Comment:
Why does this example use different placeholder values for `-p` and `-i` compared to existing ones? If they don't matter, let's make the examples consistent.

Reply:
I suggest making them consistent across the whole document unless there's a specific reason not to.

Reply:
I suppose this example is consistent with the very first examples. I think here's no point emphasizing that one may specify `-p` and `-i` parameters here